common/aws-sdk/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.bazaarvoice.emodb</groupId>
+        <artifactId>emodb-parent</artifactId>
+        <version>5.4.21-SNAPSHOT</version>
+        <relativePath>../../parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>emodb-common-aws-sdk</artifactId>
+    <packaging>jar</packaging>
+
+    <name>EmoDB AWS SDK uber-jar</name>
+
+    <dependencies>
```

Comment:
The following dependencies are a departure.  Currently Emo imports the entire "aws-java-sdk" artifact, which also imports and shades a lot of extra components, like simpledb and lightsail, which aren't used.  The following dependencies are only those Emo actually uses, and more can be added as needed.